src/foam/nanos/auth/SystemAuthService.js

```diff
@@ -38,13 +82,15 @@ foam.CLASS({
 
         // Response from group implies is cached to maintain intended performance.
         if ( group != null ) {
-          Map<String, Boolean> cache = ( Map<String, Boolean> ) getCache();
-          if ( cache.get(group.getId()) == null ) {
+          this.initCache(x);
```

Comment:
Move all of this logic to a helper method called something like isSuperGroup(group), since Joel wants the same logic accessible for other purposes. Thx